main/main.cpp

```diff
@@ -1956,6 +1956,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		OS::get_singleton()->_verbose_stdout = GLOBAL_GET("debug/settings/stdout/verbose_stdout");
 	}
 
+#if defined(MACOS_ENABLED) || defined(IOS_ENABLED)
+	setenv("MVK_CONFIG_LOG_LEVEL", OS::get_singleton()->_verbose_stdout ? "3" : "1", 1); // 1 = Errors only, 3 = Info
```

Comment:
It should be equivalent, but we have this wrapped in `OS::get_singleton()->set_environment()` so this might be worth using here for consistency.